hack/jenkins/update-jobs.sh

```diff
@@ -42,4 +42,4 @@ fi
 
 docker exec job-builder git checkout master
 docker exec job-builder git pull
-docker exec job-builder jenkins-jobs test ${config_dir}
+docker exec job-builder jenkins-jobs update ${config_dir}
```

Comment:
alternately, we should at least make this email cloud-kubernetes-team if anything changes.